changelog/db.changelog-1.68.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Sam Plackett" id="1">
+    <tagDatabase tag="v1.68.0" />
+  </changeSet>
+  <changeSet author="Sam Plackett" id="2">
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="136" />
+      <column name="schemeId" value="13" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="autoHoldCategories">
+      <column name="autoHoldCategoryId" value="137" />
+      <column name="schemeId" value="13" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="137" />
+      <column name="schemeId" value="13" />
+      <column name="name" value="Delinked Payment Hold" />
```

Comment:
should these all be capitalised when the others aren't or vice-versa?
![nitpick](https://github.com/DEFRA/ffc-pay-processing/assets/139965284/47e4f897-d4d8-4bbd-b2a7-d4a514b38a2a)

Reply:
Probably not. That's what I've been told to call it, but I'm going to overrule with your backing and make it lower case